azkaban-common/src/main/java/azkaban/executor/BaseRefreshableMap.java

```diff
@@ -60,4 +60,9 @@
   public BaseRefreshableMap<K, V> clone() {
     return (BaseRefreshableMap<K, V>) super.clone();
   }
+
+  @Override
+  public int elementCount() {
+    return this.values().stream().mapToInt(elem -> elem.elementCount()).sum();
```

Comment:
What if there is no element?

Reply:
it will return zero. The values is a map so, it could not be NPE.